src/types/device/device.ts

```diff
@@ -44,6 +45,7 @@ export interface DeviceDetail extends DeviceBase {
   current_location: LocationList | undefined;
   created_by: UserBase;
   updated_by: UserBase;
+  managing_organization: FacilityOrganization;
```

Comment:
this can be null right?

```suggestion
  managing_organization: FacilityOrganization | null;
```

Reply:
missed that, made the change.